packages/create-llama/tsconfig.json

```diff
@@ -1,11 +1,19 @@
 {
+  "extends": "../../tsconfig.json",
```

Comment:
@himself65 what's the advantage of this dependency? a disadvantage that I see is that it makes it more difficult to separate `create-llama` out of this repository (which is something we might want to do as it grows more and more)

Reply:
I originally wanted to make typescript check all ts files because some examples code was already outdated but we didn't know it. I think it's very simple to separate `create-llama`, just copy upstream tsconfig and merge it into one tsconfig.

Reply:
> I originally wanted to make typescript check all ts files because some examples code was already outdated but we didn't know it. 
Yes that's a very good idea. I just wondered if we need this tsconfig.json dependency for this?

Reply:
Not required